### ISSUE TITLE
Playground should be a handler and not middleware

### ISSUE THREAD

**Issue by BrendanBall** [graphcool/graphql-playground#614]
#### This issue pertains to the following package(s):

- [ ] GraphQL Playground - Electron App
- [ ] GraphQL Playground HTML
- [ ] GraphQL Playground
- [x] GraphQL Playground Express Middleware
- [ ] GraphQL Playground Hapi Middleware
- [ ] GraphQL Playground Koa Middleware
- [ ] GraphQL Playground Lambda Middleware

#### What OS and OS version are you experiencing the issue(s) on?
Linux

#### What version of graphql-playground(-electron/-middleware) are you experiencing the issue(s) on?
graphql-playground-middleware-express@^1.5.8

#### What is the expected behavior?
```
app.get('/graphql/playground', expressPlayground({ endpoint: '/graphql' }))
app.use('/graphql', express.json(), graphqlExpress({ schema }))
```
should work without a problem
#### What is the actual behavior?
the playground handler calls `next()` so when trying to load the playground in a browser, after loading the frontend it tries to do a `GET` on `/graphql` which is invalid. after completing the `/graphql/playground` request it should end the request.

#### What steps may we take to reproduce the behavior?
setup an example server using apollo-server-express and the handlers
```
app.get('/graphql/playground', expressPlayground({ endpoint: '/graphql' }))
app.use('/graphql', express.json(), graphqlExpress({ schema }))
```

Workaround:
```
app.get('/graphql/playground', expressPlayground({ endpoint: '/graphql' }), (req, res) => res.end())
app.use('/graphql', express.json(), graphqlExpress({ schema }))
```

_Please provide a gif or image of the issue for a quicker response/fix._

**Comment by lewebsimple** [comment]
I agree, requests to the playground should not fall through to the next handler / middleware.
Thanks @BrendanBall for the workaround !

**Comment by subuta** [comment]
Same issue exists at koa middleware

https://github.com/prisma/graphql-playground/blob/master/packages/graphql-playground-middleware-koa/src/index.ts#L24
